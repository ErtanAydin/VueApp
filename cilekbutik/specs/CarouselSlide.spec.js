import { mount, RouterLinkStub } from "@vue/test-utils";
import CarouselSlide from "@/components/CarouselSlide";

describe("CarouselSlide", () => {
  test("mounts properly", () => {
    const wrapper = CarouselSlide;
    expect(wrapper).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = mount(CarouselSlide, { 
      stubs: {
        NuxtLink: RouterLinkStub
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});