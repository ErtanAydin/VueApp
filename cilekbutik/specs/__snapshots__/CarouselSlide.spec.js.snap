// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarouselSlide renders properly 1`] = `
<transition-stub name="crossfade">
  <div class="carousel-slide"></div>
</transition-stub>
`;
